dev-ruby/gh/gh-0.14.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
+RUBY_FAKEGEM_RECIPE_DOC="rdoc"
+RUBY_FAKEGEM_RECIPE_TEST="none"
```

Comment:
There are specs and you add dependencies for them, but they are not being run? You probably want the rspec3 recipe here (and then the explicit rspec bdepend can be dropped).